doc/release-notes.md

```diff
@@ -19,7 +20,9 @@
       - cnv-cnvkit.seg.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)
       - cnv-controlfreec.tsv.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)
       - snv-consensus-plus-hotspots.maf.tsv.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)
-    - Update method to call CNV consensus as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `manta` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. And in the subsequent step, instead of ony retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. As a result of changing consensus calling criteria and adding TARGET WXS DNA sample results to `cnv-cnvkit.seg.gz` and `cnv-controlfreec.tsv.gz`, the following files were updated:
+    - Update method to call CNV consensus as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `manta` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. And in the subsequent step, instead of ony retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. Additionally, WGS and WXS samples now went through different CNV calling pipeline. CNV consensus is only called for WGS samples and hence consensus cnv file only contain WGS sampels [ticket 152](https://github.com/PediatricOpenTargets/ticket-tracker/issues/152): 
```

Comment:
```suggestion
    - Update method to call CNV consensus (WGS) as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `MantaSV` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. In the subsequent step, instead of only retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. The consensus is the overlapping region. 
```

for the future, please write one sentence per line for easier review :)  

I removed that samples went through two pipelines bc we had that in v7 as well and our filenames/data files description indicate such.

Reply:
Gotcha! Will do!